Softeq.XToolkit.ProjectInfrastructure/HttpResponce.cs

```diff
@@ -6,10 +6,22 @@ namespace Softeq.XToolkit.CrossCutting
 {
     public class HttpResponse
     {
+        private bool _isNoInternetConnection;
+
         public string Content { get; set; }
         public byte[] BinaryContent { get; set; }
         public bool IsSuccessful { get; set; }
         public HttpStatusCode StatusCode { get; set; }
+
+        public bool IsPoorConnection
+        {
+            get => _isNoInternetConnection ||
```

Comment:
need to discuss
Please avoid changes in HttpClient. Move it to 'Default Authorization' class library.

Reply:
If server will be unavailable and connection good, you will have poorconnection=true, but this is not the truth